src/main/groovy/io/seqera/wave/service/builder/DockerBuildStrategy.groovy

```diff
@@ -170,10 +161,9 @@ class DockerBuildStrategy extends BuildStrategy {
         }
 
         final img = platform.arch == "arm64"
-                ? singularityImageArm64
-                : singularityImage
+                ? buildConfig.singularityImageArm64
+                : buildConfig.singularityImage
```

Comment:
Let's move this logic into `BuildConfig`, having something like `getSingularityImage(Platform p)`

Reply:
done